src/durabletask/azext_durabletask/aaz/latest/durabletask/scheduler/_create.py

```diff
@@ -12,20 +12,19 @@
 
 
 @register_command(
-    "durabletask namespace create",
-    is_preview=True,
+    "durabletask scheduler create",
 )
 class Create(AAZCommand):
-    """Create a Namespace
+    """Create a Scheduler
 
-    :example: Create a namespace in northcentralus
-        az durabletask namespace create -g resource-group-name -n namespace-name --location northcentralus
+    :example: Create a scheduler in eastus
+        az durable-task scheduler create --resource-group testrg --scheduler-name testscheduler --location eastus --ip-allowlist "[0.0.0.0/0]" --sku "{capacity: 1, name:Dedicated, redundancyState: None}" --tags "{}"
```

Comment:
The `redundancyState` should be removed from the request. I also observed that this showed up when I generated some of the other SDKs.

In the original TypeSpec this is:
```
  @doc("Indicates whether the current SKU configuration is zone redundant")
  @visibility("read")
  redundancyState?: RedundancyState;
```

It is supposed to be read only, but somehow the compilers are treating it as a request parameter, which is false.

Reply:
Let's make sure that `redundancyState` is not in examples, and also cannot be set by users.

I just talked to @lmazuel about this in the TypeSpec Discussion chat and the `redundancyState` property really shouldn't be suggested for any requests.

See a related bug he filed: https://github.com/Azure/autorest.python/issues/2978